lib/core/viewmodels/feedback_viewmodel.dart

```diff
@@ -34,15 +46,52 @@ class FeedbackViewModel extends BaseViewModel {
     // Upload the file and create the issue
     _githubApi.uploadFileToGithub(filePath: fileName, file: file);
 
-    _githubApi.createGithubIssue(
+    final Issue issue = await _githubApi.createGithubIssue(
         feedbackText: feedbackText,
         fileName: fileName,
         feedbackType: feedbackType);
 
+    if (issue != null) {
+      setBusy(true);
+      _myIssues.add(FeedbackIssue(issue));
+      // Sort by state open first and by number descending
+      _myIssues.sort(
+          (a, b) => b.state.compareTo(a.state) * 1000 + b.number - a.number);
```

Comment:
Since we now have around 700 issues in the repository, I suggest you add 1 or 2 zeroes to `b.state.compareTo(a.state) * 1000` just to make sure issues will always be sorted the way you intend them to. Nice trick by the way 🔥